notebooks/stage/keyword_analysis.ipynb

```diff
@@ -24,6 +24,8 @@
     "import re\n",
```

Comment:
<p><code>f"{BASE_PATH}processed/..</code>  is missing a <code>/</code>   (it should be <code>f"{BASE_PATH}/processed/..</code>  (2 occurrences in the cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<pre class="ql-syntax" spellcheck="false">BASE_PATH = os.getenv("LOCAL_DATA_PATH", "../../data/")
</pre><p> </p><p>The base path has the <code>/</code>  at the end so its not needed here, but I agree its a little odd the way its written now. I'll change the base path and add the <code>/</code>  here. </p>

Reply:
Oh, I didn't notice that... :facepalm: Thank you. :smile: